arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -1182,9 +1182,10 @@ Result IResearchAnalyzerFeature::createAnalyzerPool(
   // validate analyzer name
   auto split = splitAnalyzerName(name);
 
-  if (!AnalyzerNameValidator::isAllowedName(
+  if (auto res = AnalyzerNameValidator::validateName(
           extendedNames,
-          std::string_view(split.second.data(), split.second.size()))) {
+          std::string_view(split.second.data(), split.second.size()));
```

Comment:
`split.second` should already be a string view
```suggestion
          split.second);
```